website/www/site/content/en/get-started/quickstart/python.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "Beam Quickstart for Python"
+aliases:
+  - /get-started/quickstart/
+  - /use/quickstart/
+  - /getting-started/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Apache Beam Python SDK quickstart
+
+This quickstart shows you how to run an
+[example pipeline](https://github.com/apache/beam-starter-python) written with
+the [Apache Beam Python SDK](/documentation/sdks/python), using the
+[Direct Runner](/documentation/runners/direct/). The Direct Runner executes
+pipelines locally on your machine.
+
+If you're interested in contributing to the Apache Beam Python codebase, see the
+[Contribution Guide](/contribute).
+
+On this page:
+
+{{< toc >}}
+
+## Set up your development environment
+
+### Check your Python version
+
+Apache Beam aims to work on released
+[Python versions](https://devguide.python.org/versions/) that have not yet
+reached end of life, but it may take a few releases until Apache Beam fully
+supports the most recently released Python minor version.
+
+The minimum required Python version is listed in the **Meta** section of the
+[apache-beam](https://pypi.org/project/apache-beam/) project page under
+**Requires**. The list of all supported Python versions is listed in the
+**Classifiers** section at the bottom of the page, under **Programming
+Language**.
+
+Check your Python version by running:
+
+{{< highlight >}}
+python3 --version
+{{< /highlight >}}
+
+If you don't have a Python interpreter, you can download and install it from
+the [Python downloads](https://devguide.python.org/versions/) page.
+
+If you need to install a different version of Python in addition to the version
+that you already have, you can find some recommendations in our
+[Developer Wiki](https://cwiki.apache.org/confluence/display/BEAM/Python+Tips#PythonTips-InstallingPythoninterpreters).
+
+## Clone the GitHub repository
```

Comment:
Let's move  `## Clone the GitHub repository` before ` install project dependencies`, then we can make
`## Set up your development environment` into its own section, and can reference that section in the old quickstart, instead of repeating the content there.

Reply:
Right now "Set up your dev environment" is its own section, so it's no problem to link from the old quickstart. (I think  the current flow is pretty logical, although that's partly subjective.)

Reply:
(I added the link from the old quickstart)